orchagent/port/porthlpr.cpp

```diff
@@ -468,8 +501,16 @@ bool PortHelper::parsePortFec(PortConfig &port, const std::string &field, const
         return false;
     }
 
+    const auto &override_cit = portFecOverrideMap.find(value);
+    if (override_cit == portFecOverrideMap.cend())
+    {
+        SWSS_LOG_ERROR("Failed to parse field(%s): invalid value(%s) in override map", field.c_str(), value.c_str());
+        return false;
+    }
+
     port.fec.value = cit->second;
     port.fec.is_set = true;
+    port.fec.override_fec =override_cit->second;
```

Comment:
@dgsudharsan only for none,rc,fc this will be true?

Reply:
Yes. Please refer to portFecOverrideMap